.github/workflows/publish-container.yml

```diff
@@ -0,0 +1,52 @@
+name: Publish container to GHCR
+
+permissions:
+  # Publish to GHCR.
+  packages: write
+  # Create release branch and publish release.
+  contents: write
+
+on:
+  workflow_call: {}
+
+jobs:
+  publish-container:
+    name: Publish container
+    runs-on: ubuntu-latest
+    permissions:
+      # Publish to GHCR.
+      packages: write
+    steps:
+    - name: Download build artifacts
+      uses: actions/download-artifact@v4
+      with:
+        path: out
+
+    - name: Login to GHCR
+      run: echo "${{ secrets.GITHUB_TOKEN }}" | docker login ghcr.io -u "${{ github.actor }}" --password-stdin
+
+    - name: Publish container
+      run: |
+        set -x
+        VERSION=$(<"out/version/version.txt")
+
+        DOCKER_FULL_PATH="ghcr.io/${{ github.repository_owner }}/imagecustomizer:$VERSION"
```

Comment:
side note - we should probably update the github docs to reference this acr instead (once we have one release with it out)

Reply:
Yes, will do.